### ISSUE TITLE
Negative loss function in Autoregressive

### ISSUE THREAD

**Issue by Hakim777** [tensorflow/probability#1249]
Hi

I'm using Autoregressive for density estimation, the same example that is written in this page:

https://www.tensorflow.org/probability/api_docs/python/tfp/bijectors/AutoregressiveNetwork?version=nightly

However, when I increase the dimension (event_shape) of the data that I want to find its density, from two to four for example, I start getting negative loss function!

Can anyone help me with this issue.
Thanks
